controllers/warp/warp.go

```diff
@@ -67,18 +71,56 @@ func (w *Watcher) Run(ctx context.Context) error {
 	}
 
 	for _, source := range w.configuration.Sources {
-		if strings.HasPrefix(source.Path, "/dogu") {
+		if strings.HasPrefix(source.Path, "/dogu") || strings.HasPrefix(source.Path, "dogu") {
 			w.startVersionRegistryWatch(ctx, source.Path)
 			continue
 		}
 
-		// TODO config/_global
-
-		// TODO config/
+		if strings.HasPrefix(source.Path, "/config/_global") || strings.HasPrefix(source.Path, "config/_global") {
+			w.startGlobalConfigWatch(ctx, source.Path)
+			continue
```

Comment:
Okay I understand that the warp menu config doesn't have to be changed, because it doesn't contain any values, only the keys. But nonetheless the watches don't work this way.